tests/telemetry_intake/update.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+
+# These variables may change over time
+# The path in dd-go to the rules JSON files.
+# If this script deletes the JSON files in system-tests, it's likely this is no longer valid
+SUBDIR="trace/apps/tracer-telemetry-intake/telemetry-payload/static/"
+# The "main" branch in dd-go
+BRANCH="bm1549/more-missing-configs" # This should ALWAYS be "prod" in master. Please revert changes before merging
```

Comment:
Pending (but not blocked by) https://github.com/DataDog/dd-go/pull/161868

```suggestion
BRANCH="prod"
```